ui/src/app/kvm/components/KVMHeaderForms/RefreshForm/RefreshForm.tsx

```diff
@@ -29,7 +30,7 @@ const RefreshForm = ({
       clearHeaderContent={clearHeaderContent}
       errors={errors}
       initialValues={{}}
-      modelName="KVM"
+      modelName={pluralize("KVM host", hostIds.length)}
```

Comment:
This should just be "KVM host". `ActionForm` already handles pluralising, so this results in "KVM hostss"

Reply:
Thanks!